homeassistant/components/lg_soundbar/media_player.py

```diff
@@ -53,8 +53,13 @@ def __init__(self, discovery_info):
         self._woofer_volume_max = 0
         self._bass = 0
         self._treble = 0
+        self._device = None
```

Comment:
Now that we postpone creating the device we need to remove the update in the beginning of entity addition. Ie in the call `add_entities` don't pass second argument `True`.

Reply:
Ah, good point.